tdvt/tdvt/tdvt.py

```diff
@@ -2,7 +2,7 @@
     Test driver script for the Tableau Datasource Verification Tool
 
 """
-
+import os
```

Comment:
Need to move this down to below line 14. I think PyCharm did this trying to be clever. The `import sys` here is removed from the other imports because it's needed for the python version check.